src/platform/macos.rs

```diff
@@ -257,11 +257,10 @@ impl WindowExtMacOS for dyn Window + '_ {
 }
 
 /// Corresponds to `NSApplicationActivationPolicy`.
-#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Hash)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum ActivationPolicy {
     /// Corresponds to `NSApplicationActivationPolicyRegular`.
-    #[default]
     Regular,
```

Comment:
This is a breaking change, and will prevent us from releasing it in a minor version. Either don't do this, or make the bigger change of also changing `with_activation_policy` to take an option.

Reply:
Ah you got to this too fast - didn't think you'd see my draft PR. Yeah, this is a remnant while I was trying a couple things. Was just removing it, don't wish to actually do this.